frontend/src/Store/Actions/queueActions.js

```diff
@@ -158,6 +158,12 @@ export const defaultState = {
         isSortable: true,
         isVisible: true
       },
+      {
+        name: 'added',
+        label: () => translate('AddedOn'),
```

Comment:
I think this should use the already existing `Added` translation token, which will drop the `on`.